### DISCUSSION TITLE
Can't log into Android mobile app.

### DISCUSSION THREAD

**Discussion by Hanselljlh** [immich-app/immich#11948] [Q&A]
### The bug

When logging in via the Android mobile app, it just gives an error. I can login via web browser on the mobile but not the mobile app.

![Screenshot_20240820_100852](https://github.com/user-attachments/assets/907d581c-542c-4a17-871f-d0dc226f2e3b)

![Screenshot_20240820_101112](https://github.com/user-attachments/assets/323aac62-235c-4502-9816-7875d07edb34)

### The OS that Immich Server is running on

Debain Proxmox LXC

### Version of Immich Server

v1.09.2

### Version of Immich Mobile App

v1.112.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
Update
```

### Your .env content

```Shell
Update
```

### Reproduction steps

```bash
1.Attempt login via mobile app
2.Error
3.
...
```

### Relevant log output

```shell
Unable to get user information from the server.
Error getting user information from the server [CATCH ALL]
Null check operator used on a null value
#0      UserPreferencesResponseDto.fromJson (package:openapi/model/user_preferences_response_dto.dart:81)
#1      ApiClient.fromJson (package:openapi/api_client.dart:579)
#2      ApiClient.deserialize (package:openapi/api_client.dart:158)
<asynchronous suspension>
#3      UsersApi.getMyPreferences (package:openapi/api/users_api.dart:177)
<asynchronous suspension>
#4      Future.wait.<anonymous closure> (dart:async/future.dart:524)
<asynchronous suspension>
#5      AuthenticationNotifier.setSuccessLoginInfo (package:immich_mobile/providers/authentication.provider.dart:172)
<asynchronous suspension>
#6      LoginForm.build.login (package:immich_mobile/widgets/forms/login/login_form.dart:188)
<asynchronous suspension>
```

**Comment by schuhbacca** [comment]
You need to update your server to the latest.

**Comment by lukababu** [comment]
I have the same issue, everything is up to date...

**Comment by lukababu** [comment]
I was able to solve the issue after changing my password. I use a password manager to generate a strong one, thus I was not mistyping the password as I was using the autofill function and I was able to log through the web ui.

There might be something in the Android code that prevents correctly parsing the password.

The password I used was:

7bm8kEv6FJ^eWFEaf4ihm

I also did had not change my password. I created a new account with this password and I untoggled the force change password on the first logon (I believe).

Hope this helps someone!
